api/src/main/java/org/eclipse/microprofile/health/package-info.java

```diff
@@ -21,6 +21,6 @@
 /**
  * <p>Microprofile Health</p>
  **/
-@org.osgi.annotation.versioning.Version("1.1")
+@org.osgi.annotation.versioning.Version("1.2")
```

Comment:
Shouldn't the OSGi bundle version be kept in sync with the Maven artifact version, which is now 2.2? 
Also I think you also need to update `src/main/java/org/eclipse/microprofile/health/spi/package-info.java` the same way.

Reply:
I'm not an OSGI expert @Emily-Jiang but perhaps we could find a more automatic way to update these. With Maven and resource filtering for instance...

Reply:
The whole reason to use this plugin is to work out the correct package version. The individual package version states what changes you have done comparied to the previous releases. For an instance, in this release, you have done any changes in package a, you will need to change the package version for the package a. In this way, end users will know there is no change in this package. Another scenario: if you made a binary incompatible change, the tool will tell you it is a major change. You will need to change the major version part (major.minor.micro). e.g. if the previous version is 1.0, you will have to change to 2.0. If you are not allowed to do a major release, this will be a show stopper. You will know you need to revert this change and wait for a major release train to come.

In short, the package version is _NOT_ linked to the maven artifact version but safegurad the apis inside the package. 

For more info on the package version:

> The semantics for the version parts are therefore defined as:1.major — Packages with versions that have different major parts are not compatible both for providers as well as consumers. For example, 1.2 and 2.3 are completely incompatible.2.minor — API consumers are compatible with exporters that have the same major number and an equal or higher minor version. API providers are compatible with exporters that have the same major and minor version number. For example, 1.2 is backward compatible with 1.1 for consumers but for providers it is incompatible. Consumers should therefore import [1.2,2) and providers should import [1.2,1.3).3.micro — A difference in the micro part does not signal any backward compatibility issues. The micro number is used to fix bugs that do not affect either consumers or providers of the API.4.qualifier — The qualifier is usually used to indicate a build identity, for example a time stamp. Different qualifiers do not signal any backward compatibility issues.Assuming an exported version 1.2.3.built, the following ranges provide the given semantics:[1.2,2)       Consumer importer policy: will not match when exporter goes to version 2 or later.[1.2,1.3)     Provider importer policy: will not match when exporter goes to version 1.3 or later.[1.2.3,1.2.4)  Strict importer policy: only accepts exporter of version 1.

Reply:
By the way, when MicroProfile Metrics accidently disabled this plugin and introduced a major change in the previous release, they had to respin another release to undo the major release to be included MP 3.2. Hence, it is essential to have this plugin enabled and properly version the package.

Reply:
Got it. Thx